examples/hello_world.rs

```diff
@@ -6,7 +6,7 @@ use std::{
 use std::{thread::park_timeout, time::Duration};
 
 fn main() {
-    let router = Router::new().with_endpoint("/", &["get"]);
+    let router = Router::new().with_endpoint("/", &["get"], move || async move { Ok(()) });
```

Comment:
Why does it need the double `move`?

Reply:
It's a first try, I will improve it and don't use closure anymore, it will be like :
```
async fn handler(req: HttpRequest) -> HttpResponse {
   HttpResponse::Ok(200)
}

Router::new().with_endpoint("/", &["get"], handler);
```

Reply:
Oh, it was not a critique :grin: Pure curiosity on my side - I understand why the second `move` is needed, I am instead confused when it comes to the first `move`

Reply:
Don't worry, I don't get it as a critique :)
I am totally agree with you, it was a quick copy/paste, but you reveal a possible mistake in my example here:
https://github.com/bastion-rs/bastion/blob/master/src/bastion/examples/round_robin_dispatcher.rs#L76
I have to check it ;)